Build error with Emscripten 3.1.61

Homebrew updated my Emscripten from 3.1.45 (our CI's version) to 3.1.61 and I now get a build error on my M1 Mac:
```
In file included from /Users/elalish/Code/manifold/src/sdf/src/sdf.cpp:17:
/Users/elalish/Code/manifold/src/utilities/include/hashtable.h:137:9: error: call to implicitly-deleted copy constructor of 'std::atomic<size_t>' (aka 'atomic<unsigned long>')
  137 |         used_(other.used_),
      |         ^     ~~~~~~~~~~~
/opt/homebrew/Cellar/emscripten/3.1.61/libexec/cache/sysroot/include/c++/v1/__atomic/atomic.h:36:17: note: copy constructor of 'atomic<unsigned long>' is implicitly deleted because base class '__atomic_base<unsigned long>' has a deleted copy constructor
   36 | struct atomic : public __atomic_base<_Tp> {
      |                 ^
/opt/homebrew/Cellar/emscripten/3.1.61/libexec/cache/sysroot/include/c++/v1/__atomic/atomic_base.h:145:35: note: copy constructor of '__atomic_base<unsigned long>' is implicitly deleted because base class '__atomic_base<unsigned long, false>' has a deleted copy constructor
  145 | struct __atomic_base<_Tp, true> : public __atomic_base<_Tp, false> {
      |                                   ^
/opt/homebrew/Cellar/emscripten/3.1.61/libexec/cache/sysroot/include/c++/v1/__atomic/atomic_base.h:134:3: note: '__atomic_base' has been explicitly marked deleted here
  134 |   __atomic_base(const __atomic_base&) = delete;
      |   ^
1 error generated.
```
Unfortunately Homebrew doesn't seem to provide an easy way to downgrade for this package. However, we'll need to upgrade eventually anyway, so we should fix this, or file a bug upstream if it's Emscripten's fault. Any ideas @pca006132?

--- a/.github/workflows/manifold.yml
+++ b/.github/workflows/manifold.yml
@@ -123,8 +123,8 @@ jobs:
         # setup emscripten
         git clone https://github.com/emscripten-core/emsdk.git
         cd emsdk
-        ./emsdk install 3.1.45
-        ./emsdk activate 3.1.45
+        ./emsdk install 3.1.61
+        ./emsdk activate 3.1.61
     - uses: jwlawson/actions-setup-cmake@v2
     - name: Build WASM
       run: |

--- a/src/utilities/include/hashtable.h
+++ b/src/utilities/include/hashtable.h
@@ -132,10 +132,9 @@ class HashTable {
         step_(step) {}
 
   HashTable(const HashTable& other)
-      : keys_(other.keys_),
-        values_(other.values_),
-        used_(other.used_),
-        step_(other.step_) {}
+      : keys_(other.keys_), values_(other.values_), step_(other.step_) {
+    used_.store(other.used_.load());
+  }
 
   HashTable& operator=(const HashTable& other) {
     if (this == &other) return *this;

